utils/vale/styles/Crossplane/spelling-exceptions.txt

```diff
@@ -56,4 +62,26 @@ UpperCamelCased
 user-defined
 v2
 version-specific
+Job.
+e.g.
+high-churn
+idempotency
```

Comment:
i think this list has been kept alphabetized so far - do what you will with that comment 😉